.eslintignore

```diff
@@ -0,0 +1,5 @@
+**/node_modules/*
```

Comment:
Great job on this lab in general!!! I just wanted to comment that when I pulled down, I didn't get your data/story directories and that caused some tests to fail. I made them on my end and they passed. 

I think what happened is you may have deleted your test files and then pushed the empty directories up, and Github doesn't include empty folders on their repos. 

In the future, if you want to include an otherwise empty directory into a github repo, you can just add a file called ``.gitkeep`` into that folder. :) 

Great job on this lab!